application.example.yaml

```diff
@@ -170,9 +170,12 @@ trading:
         - REP/USD
         - ZEC/USD
 
-      # The amount of fees Kraken charges. If an exchange's API supports requesting fees at runtime, the bot
+      # The amount of fees Kraken charges for long trades. If an exchange's API supports requesting fees at runtime, the bot
```

Comment:
This change to the comment is not accurate. The trade fee is applied to every trade. Long or short, margin or not.

Reply:
Aha, I didn't know that. I will fix it.